src/en/infernalvoidscans/src/eu/kanade/tachiyomi/extension/en/infernalvoidscans/InfernalVoidScans.kt

```diff
@@ -20,21 +25,26 @@ class InfernalVoidScans : MangaThemesia(
         }
         .build()
 
+    private val json by injectLazy<Json>()
+
     override fun headersBuilder() = super.headersBuilder()
         .set("Cache-Control", "max-age=0")
 
-    override val pageSelector = "div#readerarea > p > img"
-
-    override val hasProjectPage = true
+    private val titleCache by lazy {
+        val response = client.newCall(GET("$baseUrl/api/query?perPage=9999", headers)).execute()
+        val data = json.decodeFromString<SearchResponse>(response.body.string())
+        data.posts
+            .filterNot { it.isNovel }
+            .associateBy { it.slug }
+    }
```

Comment:
don't request this again, make the one in multisrc protected so it is accessable here